dockerfiles/share_task_qc_atac.dockerfile

```diff
@@ -13,20 +14,29 @@ ENV SAMTOOLS_VERSION 1.9
 RUN apt-get update && apt-get install -y \
     autoconf \
     build-essential \
+    gcc \
     git \
     libcurl4-openssl-dev \
+    libjpeg-dev \
     liblz4-dev \
     liblzma-dev \
     libncurses5-dev \
     libbz2-dev \
-    python \
+    openjdk-11-jre \
```

Comment:
Do we really need java now in the docker?
Isn't producing the same QCs but with python?

Reply:
No, the share_task_qc_atac is still using picard CollectInsertSizeMetrics to get the data which is plotted by the histogram.